partner_contact_lang/models/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    # HACK https://github.com/odoo/odoo/issues/53738
+    # TODO Remove when fixed
+    partner_lang = fields.Selection(related="lang", readonly=True)
```

Comment:
set a `string` to avoid duplicated label warnings

Reply:
Thanks, fixed.